Sources/Shared/Tailor.swift

```diff
@@ -1,5 +1,6 @@
-// MARK: - Error
+// swiftlint:disable type_name
```

Comment:
This file was throwing errors through swift lint despite following Swift 3 conventions.